ios/browser/api/url/url_origin_ios.h

```diff
@@ -82,7 +82,7 @@ NS_SWIFT_NAME(URLOrigin)
 /// Try not to use this method under normal circumstances, as it loses type
 /// information. Downstream consumers can mistake the returned GURL with a full
 /// URL (e.g. with a path component).
-@property(readonly) NSURL* url;
+@property(readonly, nullable) NSURL* url;
```

Comment:
No need to update documentation here?

Reply:
Technically not since it mentions its the same as `GURL(Serialize())` which would be invalid for opaque origins because `Serialize()` returns the string: `"null"`

I can add more info if we want though

Reply:
Added it anyways just for clarity